demos/stress/lv_demo_stress.c

```diff
@@ -32,7 +32,7 @@ static void obj_test_task_cb(lv_timer_t * tmr);
  **********************/
 static lv_obj_t * main_page;
 static lv_obj_t * ta;
-static uint32_t mem_free_start = 0;
+static size_t mem_free_start = 0;
```

Comment:
may be change %zu to LV_PRIu32 will be more proper

Reply:
[`lv_mem_monitor_t` `free_size`](https://github.com/lvgl/lvgl/blob/master/src/stdlib/lv_mem.h#L38) is also `size_t` so using it here makes sens to me